app/proto/jail.proto

```diff
@@ -61,3 +66,8 @@ message SetLocationReq {
 message AcceptCommunityGuidelinesReq {
   bool accept = 1;
 }
+
+message SetPasswordReq {
+  // the frontend should ask for the password twice and whatnot
+  string new_password = 1 [ (sensitive) = true ];
```

Comment:
What does grpc do to sensitive fields?

(Obviously unrelated to the frontend review, mainly curious about this :))

Reply:
It's used in the backend to clear those fields before they're stored in to a debugging table:

https://github.com/Couchers-org/couchers/blob/bfdfb49536d1eb2ee135a37186443f33a3fcc871/app/backend/src/couchers/interceptors.py#L185-L195